<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffResDialog</class>
 <widget class="QDialog" name="DiffResDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Difference type residual</string>
  </property>
  <widget class="QLabel" name="label_diffRes_recImg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_diffRes_OrigImg">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>390</width>
     <height>390</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_diffRes_inputs">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_diffRes_origImgLoad">
        <property name="text">
         <string>Original Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_diffRes_recImgLoad">
        <property name="text">
         <string>Reconstructed Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_diffRes_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>870</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_diffRes_saveFileName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_diffRes_ImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_diffRes_saveImg">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
